CGPs/0037.md

```diff
@@ -0,0 +1,50 @@
+# CGP [0037]: Celo Core Contracts Release 5
+
+- Date: 2021-08-12
+- Author(s): @m-chrzan
+- Status: DRAFT
+- Governance Proposal ID #: TBD
+- Date Executed: TBD
+- Discussions To: TBD
```

Comment:
Can you update the format to follow the new template? We will be updating existing CGPs to follow the new preamble https://github.com/celo-org/governance/blob/main/CGPs/cgp-template.md

also, please rename the CGP as `cgp-0037.md`